src/internal/tester/integration_runners.go

```diff
@@ -1,11 +1,13 @@
 package tester
```

Comment:
This file contains the primary logic of the change.  All other changes in the PR comply with the addition of `t` and removal of the `error` response value, from these functions.